.github/workflows/quality-report.yml

```diff
@@ -5,38 +5,39 @@ on:
     branches: [ main ]
   workflow_dispatch:
 
+
 jobs:
   build:
     name: Run Quality Report
     runs-on: macos-13
     permissions:
       contents: write
-    
+
     steps:
       - name: Add path globally
         run: echo "/usr/local/bin" >> $GITHUB_PATH
 
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # pin@v4
         with:
           lfs: 'true'
-          
+
       - name: Xcode select
         run: |
           sudo xcode-select -s /Applications/Xcode_15.1.app
-          
+
       - name: Build and Test
-        run: |
+        run: |+
```

Comment:
Is this + meant to be here?

Reply:
Its being added by the `pin-github-action` script but I will remove it.